pkg/AnewLibrary/Fields/FieldsNodeTypes.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * @license
+ * Copyright 2023 Atom54 LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+const category = 'UX';
+
+export const FieldsNodeTypes = {
```

Comment:
Is this file needed? I thought this would be replaced by the atom_manifest...

Reply:
This is just in progress.